src/lesson4/Car.java

```diff
@@ -0,0 +1,146 @@
+package lesson4;
+
+import javax.swing.*;
+
+public class Car {
+
+    private int mileage;
+    private int fuel;
+    private int speed;
+    private int coolantLevel; // Litres
+    private String color;
+    private String engine;
+
+    public Car() {
+    } // Constructor without parameters
+
+    public Car(int mileage, int fuel, int speed, int coolantLevel, String color, String engine) {
+        this.mileage = mileage;
+        this.fuel = fuel;
+        this.speed = speed;
+        this.coolantLevel = coolantLevel;
+        this.color = color;
+        this.engine = engine;
+    }
+
+    public int getMileage() {
+        return mileage;
+    }
+
+    public void setMileage(int dailyMileage) {
+        if (dailyMileage > 0 && dailyMileage <= 2000)  // 2000 daily norm for mileage
```

Comment:
2000 looks like constant, you can create class variable something like: int dailyMileageMax = 2000; and use it here